crates/holochain_state/src/query/chain_head.rs

```diff
@@ -26,8 +26,7 @@ impl Query for ChainHeadQuery {
         "
         SELECT Action.blob, Action.hash
         FROM Action
-        JOIN DhtOp ON DhtOp.action_hash = Action.hash
-        WHERE Action.author = :author AND Action.hash IS NOT NULL
+        WHERE Action.author = :author
```

Comment:
Technically we don't even need the `author = :author` constraint any more right? Because we split the authored databases up by cell so this database will only contain actions by the author in question